### DISCUSSION TITLE
Address shows after compiling and deploying instead of name or favorite number in simpleStorage

### DISCUSSION THREAD

**Discussion by wolab** [Cyfrin/foundry-full-course-cu#2069] [Q&A]
Indeed, Our instructor, Mr Patrick is doing great work.

When simpleStorage was imported to storageFactory, was compiled, and deployed, the outcome comes with address instead of the name and favoriteNumber in the simpleStorage.

Kindly explain this to me.

Thanks sir/ma

**Comment by EngrPips** [comment]
Hello @wolab, Please visit this link https://github.com/Cyfrin/foundry-full-course-cu/blob/main/how-to-ask-a-question.md to learn how to properly ask questions.

**Reply by wolab** [reply]
Thanks. Shall do that against another time

However, kindly help me out on this

**Comment by EngrPips** [comment]
I don't understand your question. But when you import  `simpleStorage` to `storageFactory,` then you will deploy `storageFactory` and not `simpleStorage` because by deploying `storageFactory,` you can then always call it to deploy different `simpleStorage` contract.
